app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -148,7 +150,12 @@ def handle_bad_request_error(err, enrollment)
     end
   end
 
-  def handle_standard_error(err, enrollment)
+  def handle_parsing_err(err, enrollment)
+    message = err.message.split(':')[1].lstrip
```

Comment:
might be specific enough to warrant a comment about what the message looks like/how we're getting here. maybe something like "message is at index 1 of array"? (that's what i'm assuming is the case)

Reply:
oh - is this specifically for a nil message? if so could include that in the comment

Reply:
@svalexander I'm inferring that this change was made to fix a test, so I proposed [an alternative solution](https://github.com/18F/identity-idp/pull/7057#discussion_r984138149) that would leave this part of the original code intact.